### PR TITLE
Dependency upgrades

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1410)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -17,7 +17,7 @@ digraphVersion = 1.0
 disklrucacheVersion = 2.0.2
 hamcrestCoreVersion = 1.3
 jacksonVersion = 2.9.8
-jcommanderVersion = 1.74
+jcommanderVersion = 1.75
```
Comment (mnonnenmacher) [inline gradle.properties]: I get `java.lang.NoClassDefFoundError: com/beust/jcommander/JCommander` when I try this branch locally, I saw the same error on CI. The JAR of 1.75 contains basically the same files as 1.74, except for an empty `kobaltBuild` directory and some OSGI info in the manifest. The class files have the same sizes, but are not identical.

Comment (mnonnenmacher) [reply gradle.properties]: The manifest of 1.75 is also broken:
https://github.com/cbeust/jcommander/commit/85cb6f7217e15f62225185ffd97491f34b7b49bb#commitcomment-32810824

Comment (sschuberth) [reply gradle.properties]: I saw that comment, but interestingly a local build succeeded for me, so I wanted to wait until I also see it fail on CI myself.

Comment (sschuberth) [reply gradle.properties]: [Confirmed](https://travis-ci.com/heremaps/oss-review-toolkit/builds/106141348#L733), I've dropped the JCommander upgrade commit.

**Comment by sschuberth** [comment]
Merging, again ignoring the ongoing AppVeyor failure.

**Pull request merged**